backend/hubs/models/hub.py

```diff
@@ -367,3 +366,94 @@ class Meta:
 
     def __str__(self):
         return "%s" % (self.name)
+
+class HubThemeColor(models.Model): 
+    main = models.CharField(
+        help_text="main color",
```

Comment:
Which formats can we use here? hex and rgb? It would be helpful to specify that in the help text, for example like in this suggestion
```suggestion
        help_text="main color (use hex code, e.g. #FFF)",
```